### DISCUSSION TITLE
Co-Op

### DISCUSSION THREAD

**Discussion by LAP115** [LivingFray/HaloCEVR#53] [Q&A] [answered]
Is there any way for an  online co op campaign to work  with this mod? If so how can that happen?

**Comment by HoriZon-HZN** [comment, marked as answer]
It never supported co-op on PC so would need a lot of work to add even if possible.

**Reply by Daydricoldsmith** [reply]
Wouldn't co-op just work through mcc? Since halo ce has two player online co-op?

**Reply by LAP115** [reply]
this edition of halo is not on mcc
